pyronear/datasets/openfire.py

```diff
@@ -101,38 +116,55 @@ def _check_exists(self, train=True):
         else:
             return self._root.joinpath(self._processed, self.test_file).is_file()
 
-    def download(self, threads=None, valid_pct=None):
-        """Download the OpenFire data if it doesn't exist in processed_folder already.
-
+    def create_directories(self, max_files=None):
+        """
+        Create the directory structure to hold the images and training_file/test_file.
+        Download the json file with annotations, add 'fname' with local filenames
+        
```

Comment:
There is a trailing whitespace here according to Codacy.

Reply:
Fixed (codacy is annoying, this is just documentation)